src/week1/day0/css/gallery.css

```diff
@@ -0,0 +1,86 @@
+/* Small screens (default) */
+html { 
+    font-size: 100%; 
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+/* Medium screens (640px) */
+@media (min-width: 40rem) { 
+  html { font-size: 112%; }
+}
+
+/* Large screens (1024px) */
+@media (min-width: 64rem) { 
+  html { font-size: 120%; }
+}
+
+@media (max-width: 350px) {
+    .container > h1 {
+        font-size: 3rem;
+    }
+}
+
+*, *:before, *:after {
+    -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+.container {
+    margin: 0 auto;
+    max-width: 48rem;
+    width: 90%;
+  }
+
+.wrapper {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+
+h1 {
+    text-align: center;
+    font-weight: bolder;
+    font-size: 4rem;
+}
+
+.box {   
+    height: 15rem;
+    width: 15rem;
+    margin: 10px auto 10px auto;
+    border: #000000 2px solid;
+}
+
+img {
+    width: 100%;
+    height: inherit;
+    padding: 5px;
+}
+
+.title {
+    background-color: rgba(128, 128, 128, 0.7);
+    position: relative;
```

Comment:
- Remember to use "position: relative" in your "container or parent" element. E.g: `<div class="box">`. After that, you will have to set "position: absolute" in your child element, in this case `<div class="title">`. It will be better.